setup.py

```diff
@@ -120,6 +122,7 @@ def run_setup(with_binary=True):
         extras_require={
             'docs': requirements['docs'],
         },
+        license='Free for non-commercial use',
```

Comment:
I don't think this is correct:
https://docs.python.org/3/distutils/setupscript.html#additional-meta-data

Reply:
Fixed.